glean_parser/lint.py

```diff
@@ -262,6 +262,36 @@ def check_expired_metric(
         yield ("Metric has expired. Please consider removing it.")
 
 
+def check_redundant_ping(
+    pings: Iterable[pings.Ping]
```

Comment:
This check will receive one ping object at a time, so this should be `ping: pings.Ping`.
Additionally we always pass in the configuration object as well, so it takes a second argument.
The full thing should read

```
def check_redundant_ping(
    ping: pings.Ping, parser_config: Dict[str, Any]
) -> LintGenerator:
```

Reply:
I didn't understood the use of the second argument that is why I didn't had, now that you said we always pass this argument I included it.

Reply:
It's not used in your code, but because of how we call these functions it's always passed in case the function _does_ need it.